Mathlib/Topology/Algebra/UniformMulAction.lean

```diff
@@ -116,6 +116,38 @@ instance UniformGroup.to_uniformContinuousConstSMul {G : Type u} [Group G] [Unif
     [UniformGroup G] : UniformContinuousConstSMul G G :=
   ⟨fun _ => uniformContinuous_const.mul uniformContinuous_id⟩
 
+section Ring
+
+variable {R β : Type*} [Ring R] [UniformSpace R] [UniformAddGroup R] [ContinuousMul R]
+  [UniformSpace β] {f : β → R}
+
+theorem UniformContinuous.const_mul' (hf : UniformContinuous f) (a : R) :
```

Comment:
Similarly here. What is the `'` doing?

Reply:
The unprimed `UniformContinuous.const_mul` is added in the PR as well. That one has the strong assumption of `UniformGroup` on the domain, while this primed version requires a totally different assumption: `UniformContinuousConstSMul R R` -- on the codomain.

How do you suggest we differentiate the two sets?